source/2019-08-16-ember-3-12-released.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Ember 3.12 Released
 author: Kenneth Larsen
-tags: Releases, 2019, 3, 3.12
+tags: Releases, 2019, 3, 3.12, 3.x
```

Comment:
It looks like the other releases the tag is actually `Version 3.x` 🤔 for example https://raw.githubusercontent.com/ember-learn/ember-blog/master/source/2019-07-15-ember-3-11-released.md

Can you verify before we merge?

Reply:
Doh, good point; I failed at reading. Updating!